codeHF/workflows.yml

```diff
@@ -234,6 +234,40 @@ workflows:
     dependencies:
       - o2-analysis-hf-candidate-selector-d0
 
+  o2-analysis-je-jet-finder-hf-data: &jethf
+    executable: o2-analysis-je-jet-finder-hf
+    dependencies: o2-analysis-hf-task-d0
```

Comment:
I don't think the D0 task is needed to run this. Should this be the selector instead?

Reply:
Some dependencies actually seem missing.